queue/queue_test.go

```diff
@@ -22,6 +24,42 @@ const (
 	testCfgPath       = "../testenv/config.json"
 )
 
+func TestParseQueueName(t *testing.T) {
+	qm := Manager{}
+	if err := qm.parseQueueName(IpfsKeyCreationQueue); err != nil {
+		t.Fatal(err)
+	}
+	host, err := os.Hostname()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if ok := strings.Contains(qm.QueueName.String(), host); !ok {
+		t.Fatal("failed to properly parse queue name")
+	}
+}
+
+func TestParseQueueFull(t *testing.T) {
+	cfg, err := config.LoadConfig(testCfgPath)
+	if err != nil {
+		t.Fatal(err)
+	}
+	loggerConsumer, err := log.NewLogger("", true)
```

Comment:
I recently learned about this: https://godoc.org/go.uber.org/zap/zaptest#NewLogger

It logs all logs to `t.Log` instead, so only logs in the specific failing test shows up, not urgent now but at least for the API tests it will help with debugging

Reply:
interesting, that will be very helpful